inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/clamp_fq.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include <gna/gna_config.hpp>
+
+#include "subgraph_tests/clamp_fq.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace SubgraphTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP16, InferenceEngine::Precision::FP32
+};
+
+
+using ConfigType = std::map<std::string, std::string>;
+const ConfigType configFP32 = {
+        {"GNA_DEVICE_MODE", "GNA_SW_FP32"},
+};
+
+const ConfigType configSWExact = {
+        {"GNA_DEVICE_MODE", "GNA_SW_EXACT"},
+        {"GNA_COMPACT_MODE", "NO"}
+};
+
+const std::vector<std::pair<std::string, ConfigType>> gnaQuantModes = {
+        {"sw_fp32", configFP32},
+        {"sw_exact", configSWExact}
+};
+
+const std::vector<std::vector<size_t>> inputShapes = {
+        {1, 250},
+        {1, 640},
+        {1, 1024}
+};
+
+const std::vector<size_t> level = {65535};
+const std::vector<std::vector<float>> inputParams = {
+                                                     {-16, 16, 1},
```

Comment:
I still don't understand. This test should fail if i.e. input range is <-40, 40> and clamp minmax is set i.e. to <-20, 20>. Values outside of <16, 16> will be clamped by hardcoded scale factor = 2048.

Reply:
If clamp without FQ than this case will be failed.